package.json

```diff
@@ -1,19 +1,23 @@
 {
   "name": "@mapbox/s3scan",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Streaming operations on S3 objects",
-  "main": "index.js",
+  "main": "dist/lib/index.js",
   "engines": {
     "node": "^6.0.0"
   },
   "scripts": {
-    "test": "eslint lib index.js && tape test/*.test.js",
+    "prebuild": "node -p \"'export const LIB_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
+    "build": "npx tsc",
```

Comment:
Wouldn't just `tsc` instead of `npx tsc` be enough? I believe tsc comes from the `typescript` dependency.

Reply:
If you run `tsc` through `npm run`, then it will automatically use `tsc` from the local dependencies, which means you can drop `tsx` here. I've recently verified this in another project.